FunctionTesting/Tests/ro/sci/bv/CalculateMax/CalculateMaxTest.java

```diff
@@ -0,0 +1,48 @@
+package ro.sci.bv.CalculateMax;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+/**
+ * CalculateMaxTest checks if the method getMax returns the maximum number between some values
+ *
+ * @author Bogdan Butuza
+ */
+class CalculateMaxTest {
+
+    CalculateMax max = new CalculateMax();
+
+    /**
+     * Testing if the expected result is reached
+     */
+    @Test
+    public void getMaxEquals() {
+        int expected = max.getMax(24,94);
+        Assertions.assertEquals(expected, 94);
+    }
+
+    /**
+     * Testing if between 2 similar values, it returns one of them
+     */
+    @Test
+    public void getMaxSameNumbers() {
+        Assertions.assertEquals(max.getMax(9,9),9);
+    }
+
+    /**
+     * Testing if the maximum between three values is reached
+     */
+    @Test
+    public void getMaxThreeNumbersEquals() {
+        Assertions.assertEquals(max.getMax(max.getMax(-64,82),20),82);
+    }
+
+    /**
+     * Supposing the expected result is not reached
+     */
+    @Test
+    public void getMaxNotEquals() {
+        int actual = max.getMax(105,230); // 230
+        Assertions.assertNotEquals(max.getMax(max.getMax(94,-10),105),actual);
```

Comment:
what behavior are you trying to test here? I see that you getMax(105,230) -> 230... but you're comparing that with max from other 3 numbers... why?

a valid test would be to compare the result from getMax(105,230) to 230. to be sure it works. you did that in the first test with 24 and 94. So this test does not actually help with keeping the code correct